client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py

```diff
@@ -125,27 +126,29 @@ def process(self, instance):
             frame_end = int(frame_end)
             handle_end = int(handle_end)
             data_to_update["sg_last_frame"] = frame_end + handle_end
-        
-        # Add a few extra fields from OP to SG version
-        for op_field, sg_field in self.fields_to_add.items():
-            field_value = instance.data.get(op_field) or context.data.get(op_field)
+
+        # Add a few extra fields from AY to SG version
+        for ay_field, sg_field in self.fields_to_add.items():
+            field_value = (
+                instance.data.get(ay_field) or context.data.get(ay_field))
             if field_value:
-                # Break sg_field tuple into whatever type of data it is and its name
+                # Break sg_field tuple into whatever type of data it
+                # is and its name
                 type_, field_name = sg_field
 
                 data_to_update[field_name] = type_(field_value)
 
-        # Add version objectId to "sg_op_instance_id" so we can keep a link
+        # Add version objectId to "sg_ayon_id" so we can keep a link
         # between both
-        version_entity = instance.data.get("versionEntity", {}).get("_id")
+        version_entity = instance.data.get("versionEntity", {}).get("id")
```

Comment:
Note for future cleanup PR this variable should not be called `versoin_entity`.

Reply:
fixed in f9ddbb37e370bbd1b6c7073afe4ad385858d0fe4